sdk/internal/recording/recording.go

```diff
@@ -473,11 +473,35 @@ func defaultOptions() *RecordingOptions {
 	}
 }
 
-func (r RecordingOptions) hostScheme() string {
+func (r RecordingOptions) RouteURL(t *testing.T, rawReq *http.Request) {
```

Comment:
To me, "route" suggests finding a handler, not modifying the request. ReplaceURL?

Reply:
That's a reasonable thought, ReplaceURL isn't obvious what is being replaced. RouteURLToProxy? It's longer but it's more clear whats hapening

Reply:
I still shy away from "route", perhaps just because of past web dev experience, but setting that aside it doesn't suggest modifying the request. What's changing is the authority so, some verb + authority? ReplaceAuthority? SubstituteAuthority?

Reply:
Changed to ReplaceAuthority which I believe is more clear